CHANGELOG.rst

```diff
@@ -5,6 +5,15 @@ Change Log
    This file loosely adheres to the structure of https://keepachangelog.com/,
    but in reStructuredText instead of Markdown.
 
+2023-06-06
+**********
+
+Fixed
+=====
+
+- Corrected all security email address to security@axim.org.
```

Comment:
I believe we should be using security@openedx.org instead.

Reply:
This is the email address on https://openedx.org/community/connect/, but I'll use whatever is the right address.

Reply:
Here's an issue on the wg-security issue tracker: https://github.com/openedx/wg-security/issues/15

They still have a lot of places to update, and that page is one of them, unfortunately.

Reply:
@feanil ?  What is the canonical address to use?

Reply:
OK, I changed it to openedx.org